did/plugins/github.py

```diff
@@ -217,6 +218,16 @@ def fetch(self):
         self.stats = [
             Issue(issue) for issue in self.parent.github.search(query)]
 
+class ReleasesCreated(Stats):
+    """ Releases created """
+
+    def fetch(self):
+        for project in self.user.projects:
+            log.info("Searching for releases created by {0} in {1}".format(self.user, project))
+            query = "repos/{0}/releases?q=author:{1}+created:{2}..{3}".format(
+                project, self.user.login, self.options.since, self.options.until)
+            self.stats = [
+                Releases(release) for release in self.parent.github.search(query)]
```

Comment:
Is seems that you need to define the `Release` class as well. It could be similar to the `Issue` class above.